catroid/build.gradle

```diff
@@ -50,20 +50,6 @@ apply from: 'gradle/standalone_apk_tasks.gradle'
 check.dependsOn 'checkstyle'
 check.dependsOn 'pmd'
 
-def appId = 'org.catrobat.catroid'
-def appName = '@string/app_name'
```

Comment:
I liked that a lot! :crying_cat_face:
Any chance to get this running with these variables?

Reply:
I plan to add it back again later, see also CAT-2400 on jira.

Though right now it is broken, so I fear the changes have to go.
In the future more PR jobs will ensure that issues like these are found earlier.